specs/0108-community-health-dashboard.md

```diff
@@ -0,0 +1,49 @@
+
+There's no historic data to provide insights into how the community health is developing. Missing is the information on how completion and volume of unreviewed strings are changing over time. It's not clear what share of translations was copied from the Machinery. We also don't know the ratio of translations approved without a peer review.
+
+# Out of scope
+
+At this stage we’re only interested in presenting data we already collect in Pontoon. We're primarily focusing on exposing data to team managers; project managers will be able to obtain data across all teams if needed, e.g. by using scripts and the API.
+
+# Feature explanation
+
+A new "Insights" tab is made available on the Team page, positioned next to the "Contributors" tab. It consists of several sections, presenting data for the period of the last 12 months. Each section is described below. Fore more details about the design, see the [Mockup](#mockup) section.
+
+## Active users
+
+Pie charts showing ratios of active vs. all managers, reviewers and contributors. See the existing scripts ([1](https://github.com/flodolo/scripts/blob/954fa85/pontoon/active_contributors.py), [2](https://github.com/flodolo/scripts/blob/954fa85/pontoon/list_reviewers_with_contribution_stats.py)) for criteria of defining active users and for details on how to retrieve data.
+
+## Unreviewed suggestion lifespan
+
+A line chart showing time before suggestion gets approved or rejected. See the [existing script](https://github.com/flodolo/scripts/blob/954fa85/pontoon/unreviewed_suggestions_lifespan.py) for details on how to retrieve data. A tooltip showing the exact data at a given time appears when hovering over a chart.
+
+## Translation activity
+
+A combination of a line chart showing completion over time and a column chart showing approved translation submissions and source string additions over time in two separate columns. The translation submission column is a stack of two columns - human translations (`Translation.machinery_sources` not set) and Machinery translations (`Translation.machinery_sources` set). A tooltip showing the exact data at a given time appears when hovering over a chart.
+
+See the [existing script](https://github.com/flodolo/scripts/blob/954fa85/pontoon/self_approval_ratio.py) for calculating self-approval ratio.
+
+## Review activity
+
+A combination of a line chart showing the number of unreviewed strings over time and a column chart showing review actions and suggestion submissions over time in two separate columns. The review actions column is a stack of three columns - peer-approvals, self-approvals and rejections. A tooltip showing the exact data at a given time appears when hovering over a chart.
```

Comment:
The graph on stage is missing the right axis at this point.

Reply:
That's by design. See https://github.com/mozilla/pontoon/pull/1716#discussion_r512563932.

Reply:
I missed that part. At this point, why not have a group with 3 bars? The line is clearly following a different scale, so it's even more confusing than trying to figure out which graph belongs to which axis.

Reply:
I thought about that, too. The thing is that the amount of Unreviewed suggestions is a result of items in bars (some sort of summary), so I think it deserves a special attention. The same is true for the Completion line in the Translation Activity chart.

Should I put the secondary Y axis back? That would be consistent with the Translation Activity chart.

Reply:
> Should I put the secondary Y axis back? That would be consistent with the Translation Activity chart.

I think so, otherwise the graphic looks wrong (vs confusing) ;-)

Reply:
Updated the screenshot and the stage.